Base/QTGUI/qSlicerSaveDataDialog.cxx

```diff
@@ -67,7 +67,7 @@ namespace
 }
 
 //-----------------------------------------------------------------------------
-qSlicerFileNameItemDelegate::qSlicerFileNameItemDelegate( QObject * parent )
+qSlicerFileNameItemDelegate::qSlicerFileNameItemDelegate(QObject* parent)
```

Comment:
Putting the const and spacing changes into a different commit would make it easier to review the changes specific to this PR.

Reply:
Has actually anything changed in the dialog (other than style)? I've already spent 5 minutes with reviewing changes, but all that I noticed were trivial changes and don't see any functional difference. Maybe I just did not pay enough attention, but now I don't want to spend another 5-10 minutes with reading through these changes again. I agree with Steve that you could submit these style changes in another code pull request so that we can focus on the functional changes here.

Reply:
Yes, I waded through a lot before I found the substantive changes - the message log text is appended to the error dialogs if the data doesn't save.  I'd include the line numbers here but that would mean I need to hunt for them again 😄

Reply:
I will certainly do that next time.  Would it help if I put in another commit to this pull request that undoes these spacing changes?

Reply:
It looks like we will have another couple rounds of discussion, so yes, if you can revert the spacing changes and put them in a different PR that would make things easier.

Reply:
Commit https://github.com/Slicer/Slicer/pull/5045/commits/e350b0e2c61beb6973ef1e2e5afbf3dfb9da4103 reverts the const, Self::, this->, and spacing changes, leaving only those changes that are most specific to the present pull request.